modules/oe_paragraphs_contact/config/install/field.field.paragraph.oe_contact.field_oe_title.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_oe_title
+    - paragraphs.paragraphs_type.oe_contact
+id: paragraph.oe_paragraph_contact.field_oe_title
+field_name: field_oe_title
+entity_type: paragraph
+bundle: oe_contact
+label: Heading
+description: ''
+required: false
+translatable: true
```

Comment:
Translatable storage was in the ticket only.

Reply:
Indeed but following the requirements [page](https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?pageId=616366260), the field should be translatable and as per Contact field notes, also that one should be translatable. Checking the other fields it seems we're setting them translatable when needed including the field config, not only the storage.